docs/run_test/platform.md

```diff
@@ -0,0 +1,44 @@
+# Platform
```

Comment:
Does it mean run tests on different platform? The only word "platform" is confusing.